src/integration/pages/mostReadPage/ampTests.js

```diff
@@ -0,0 +1,5 @@
+import { runCoreAmpTests } from '../../common';
+
+export default () => {
+  runCoreAmpTests();
+};
```

Comment:
Shouldn't we be running analytics tests here?

Reply:
I didn't see this comment til now 🙈 I will add the analytics test now 👍